libs/@guardian/source-react-components/src/choice-card/ChoiceCard.stories.tsx

```diff
@@ -82,7 +69,10 @@ IconDefaultTheme.args = {
 
 // *****************************************************************************
 
-export const DarkTheme = Template.bind({});
-DarkTheme.args = {
-	theme: choiceCardThemeDark,
+export const CustomTheme: StoryFn<ChoiceCardProps> = Template.bind({});
+CustomTheme.args = {
+	theme: {
+		backgroundUnselected: 'black',
+		backgroundSelected: 'hotpink',
+	},
```

Comment:
is the `theme: choiceCardDark` better if we consider this as part of the documentation for this component. It is the "recommended" way of theming

Reply:
That makes sense. Happy to go with that. I had something similar before, we could change the colours to come from the palette because that may be more how we want the prop used.
```suggestion
CustomTheme.args = {
	theme: {
		backgroundUnselected: palette.neutral[0],
		backgroundSelected: palette.lifestyle[500],
	},
```

Reply:
yeah i think we do want that, but part of me felt that because right now you _can_ do the 'wrong' thing, we should have a story that does that, then have it break when we fix it. maybe that's the wrong approach.

but maybe we shoudl fix the types in foundations so you can't?